package.json

```diff
@@ -109,6 +110,7 @@
     "postcss-loader": "^2.1.2",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "react-emotion": "^9.1.2",
```

Comment:
Both the `emotion` and `react-emotion` libraries are required for `styled-componets`-like syntax. Checkout how the 2 files differ!! 🎉